cg/cli/workflow/balsamic/base.py

```diff
@@ -291,25 +298,26 @@ def run(context, dry, run_analysis, config_path, priority, email, case_id):
         config_path = Path.joinpath(root_dir, case_id, case_id + ".json")
 
     # Call Balsamic
-    command_str = f" run analysis" f" --account {slurm_account}" f" -s {config_path}"
+    command_str = ""
 
     if run_analysis:
         command_str += " --run-analysis"
 
     if email:
         command_str += f" --mail-user {email}"
 
+    if analysis_type:
+        command_str += f" -a {analysis_type}"
```

Comment:
Lets use full option name here:

```suggestion
        command_str += f" --analysis-type {analysis_type}"
```